tests/testthat/helper-batch_data.R

```diff
@@ -14,6 +14,12 @@ batch_data <- function(key, model_version = "model_test", type = "record_id",
   if (add_column) {
     ret$new_col <- 1
   }
+  if (isTRUE(na)) {
+    ret$y <- NA_character_
+  }
+  if (identical(na, "all")) {
+    ret[, colnames(ret) != "id"] <- NA_character_
+  }
```

Comment:
Should be another way to do it! I want to make sure I'm testing multiple columns though.